app/models/repository.rb

```diff
@@ -37,8 +37,8 @@ def by_url(url)
       Repository.find_by(external_url: url)
     end
 
-    def remove(repository)
-      return unless repository.custom?
+    def remove_if_custom(repository)
+      return false unless repository.custom?
```

Comment:
You can omit the `false` here and return nil: `repository.destroy! unless repository.custom?`